framework/generated/khronos_generators/khronos_base_decoder_body_generator.py

```diff
@@ -213,7 +192,7 @@ def make_decode_invocation(self, value, preamble, main_body, epilogue):
                     is_class and value.pointer_count > 1
                 ):
                     if type_name in self.base_header_structs.keys():
-                        base_type_name = self.gen_child_var_name(value.base_type)
+                        base_type_name = self.makeSimpleVarName(value.base_type)
```

Comment:
I wonder what we should unify the name rule to pothole_case(make_simple_var_name)? or is it better to use dromedaryCase?

Reply:
@bradgrantham @MarkY-LunarG -- is there a standard all new code (including new functions for this refactor) should adhere to.  Right now it's a mix.

Reply:
I don't have a preference but we state PEP-0008 in CONTRIBUTING.md.  PEP-8 appears to prefer pothole/snake case for functions and variables.  Am I reading it right?  https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
I was following the Khronos standard of naming which uses lower-camel-case for function names and lower-snake-case for variables.  I honestly missed the whole PEP-0008 statement.  I'm ok with reverting it back.

Reply:
For example, `beginFile`, `endFile`, etc.

Reply:
There are some functions with a specified naming from from the reg.py calls.  We can PEP-0008 everything else in the new code.

Having function and variable naming match is somewhat odd until one considers how lambda heavy idiomatic/pythonic coding is.

I'll review that new def's match the standard.